src/adb/reboot.js

```diff
@@ -0,0 +1,7 @@
+let debug = console.log.bind(console, '[adb/pull]');
+let exec = require('mz/child_process').exec;
+
+module.exports = function reboot() {
+  debug('adb reboot');
```

Comment:
It may seem overkill for the moment, but I think using `shell` would be beneficial in the long run:

```js
return adb.shell('reboot');
```

Also `reboot` is a tricky command because even though the device gets rebooted, we only await the completion of the command. The device is still rebooting and you don't know if you can issue more commands. I'd probably couple this with a `wait-for-device`:

```js
await adb.shell('reboot');
return exec('adb wait-for-device');
```